application/server/server_test.go

```diff
@@ -1070,18 +1072,50 @@ func runSmokeTest(t *testing.T, repo string, commit string, file1 string, file2
 		if !ok || codeIssueData["hasAIFix"] == false {
 			continue
 		}
-		call, err := loc.Client.Call(ctx, "workspace/executeCommand", sglsp.ExecuteCommandParams{
+		call, e := loc.Client.Call(ctx, "workspace/executeCommand", sglsp.ExecuteCommandParams{
 			Command:   snyk.CodeFixDiffsCommand,
 			Arguments: []any{uri.PathToUri(scanParams.FolderPath), uri.PathToUri(issue.FilePath), issue.Id},
 		})
-		assert.NoError(t, err)
+		assert.NoError(t, e)
 		var unifiedDiffs []code.AutofixUnifiedDiffSuggestion
-		err = call.UnmarshalResult(&unifiedDiffs)
-		assert.NoError(t, err)
+		e = call.UnmarshalResult(&unifiedDiffs)
+		assert.NoError(t, e)
 		assert.Greater(t, len(unifiedDiffs), 0)
 		// don't check for all issues, just the first
 		break
 	}
+
+	checkFeatureFlagStatus(t, &loc)
+}
+
+func checkFeatureFlagStatus(t *testing.T, loc *server.Local) {
+	t.Helper()
+
+	call, err := loc.Client.Call(ctx, "workspace/executeCommand", sglsp.ExecuteCommandParams{
+		Command:   snyk.GetFeatureFlagStatus,
+		Arguments: []any{"bitbucketConnectApp"},
+	})
+
+	if err != nil {
+		t.Fatal("FeatureFlagStatus command failed", err)
+	}
+
+	if err := call.Error(); err != nil {
+		log.Warn().Err(err).Msg("Received an error response from FeatureFlagStatus command")
+	}
+
+	log.Debug().Str("FeatureFlagStatus", call.ResultString()).Msg("Command result")
+
+	var result map[string]interface{}
```

Comment:
suggestion: use `any` instead of `interface` - same thing but easier on (my) eyes

Reply:
I read this post once: https://bitfieldconsulting.com/golang/map-string-interface and I understood that `any` came later in the language and it was more Go semantic to use `map[string]interface{}`. Also they talk about readability and simplicity so I'll change it to `any` then! 🙌